cmake/Modules/Packages/USER-COLVARS.cmake

```diff
@@ -21,6 +23,11 @@ set_target_properties(colvars PROPERTIES OUTPUT_NAME lammps_colvars${LAMMPS_MACH
 target_include_directories(colvars PUBLIC ${LAMMPS_LIB_SOURCE_DIR}/colvars)
 target_link_libraries(lammps PRIVATE colvars)
 
+if(COLVARS_DEBUG)
+  target_compile_definitions(colvars PRIVATE -DCOLVARS_DEBUG)
+  target_compile_definitions(lammps PRIVATE -DCOLVARS_DEBUG) # For the proxy
```

Comment:
This is wrong, just make the define `PUBLIC` in the line above and remove this line.

Reply:
Thanks. Fixed in 85d36ad.